src/main/resources/public/content/github-markdown.css

```diff
@@ -1,985 +1,735 @@
 .markdown-body .octicon {
-  display: inline-block;
-  fill: currentColor;
-  vertical-align: text-bottom;
+    display: inline-block;
+    fill: currentColor;
+    vertical-align: text-bottom
 }
 
```

Comment:
is this now an autogenerated file? Because a lot of line were deleted

Reply:
I ran it through a css cleaner